Solutions/Digital Shadows/Package/mainTemplate.json

```diff
@@ -0,0 +1,764 @@
+        "queryFrequency": "PT5M",
+        "queryPeriod": "PT6M",
+        "severity": "Medium",
+        "suppressionDuration": "PT1H",
+        "suppressionEnabled": false,
+        "triggerOperator": "GreaterThan",
+        "triggerThreshold": 0,
+        "eventGroupingSettings": {
+          "aggregationKind": "AlertPerResult"
+        },
+        "customDetails": {
+          "impact": "impact",
+          "mitigation": "mitigation",
+          "triage_id": "[variables('_triage_id')]",
+          "severity": "EventOriginalSeverity",
+          "description": "description",
+          "status": "status"
+        },
+        "alertDetailsOverride": {
+          "alertSeverityColumnName": "EventSeverity",
+          "alertDisplayNameFormat": "Digital Shadows - {{EventMessage}}",
+          "alertDescriptionFormat": "{{description}}\n\n{{impact}}\n\n{{mitigation}}"
+        },
+        "incidentConfiguration": {
+          "groupingConfiguration": {
+            "lookbackDuration": "7d",
+            "reopenClosedIncident": true,
+            "enabled": true,
+            "groupByCustomDetails": [
+              "triage_id"
+            ],
+            "matchingMethod": "Selected",
```

Comment:
@NikTripathi , Please remove the comma at the end of this line. This is causing error when we try to deploy it as ARM template on Azure.